### ISSUE TITLE
missing WakaTime plugin

### ISSUE THREAD

**Issue by gandarez** [notepad-plus-plus/nppPluginList#77]
My request was just merged into master but I can't find WakTime plugin for x64 and for x86 is outdated

https://github.com/notepad-plus-plus/nppPluginList/commit/ea356fff8f429041f14779d8a89d684ee19c575c

**Comment by MetaChuh** [comment]
@gandarez 

the plugins admin NppPluginList is currently only deployed with an update of notepad++.

WakaTime 4.1.0 will be available in NppPluginList 1.0.9, which will be shipped with the next notepad++ release, if no schedules collide.

many thanks for your commit and your patience.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by gandarez** [comment]
tks
